modules/common/affinity/affinity.go

```diff
@@ -53,4 +58,46 @@ func DistributePods(
 			},
 		},
 	}
+	// patch the default affinity Object with the data passed as input
+	if overrides != nil {
+		patchedAffinity, _ := toCoreAffinity(defaultAffinity, overrides)
```

Comment:
can we ignore the error?

Reply:
mmm we could backpropagate the resulting error, I agree

Reply:
Done